pyproject.toml

```diff
@@ -20,7 +20,9 @@ dependencies = [
     "pydantic~=1.0",
     "loguru",
     "PyYaml>=6",
-    "typing_extensions>=3.10"
+    "typing_extensions>=4.0.0",
+    "igraph>=0.11.5",
```

Comment:
can you put all that in optional dependencies as it's not strictly needed for tawazi to work please ?

Reply:
fixed